.travis.yml

```diff
@@ -32,7 +32,7 @@ matrix:
         - "PYTHON=python3"
       before_install:
         - brew update
-        - brew install python3
+        - brew outdated python || brew upgrade python
```

Comment:
`brew outdated` returns with success if python isn't installed. Perhaps this line should be `brew install python || brew upgrade python`?

Reply:
python is installed on macOS travis-ci images.
Otherwise, I can change this by
```
brew install python || brew outdated python || brew upgrade python
```

Reply:
Sorry should be 
```
brew install python && ( brew outdated python || brew upgrade python )
```
as install returns success even if already installed.

Reply:
well, I just did a test and it seems the behavior of brew install is not really consistent.
brew install python exits with 0 on my laptop (python is already installed). It exits with error code on travis-ci.  
https://discourse.brew.sh/t/what-exit-code-is-brew-install-supposed-to-return-when-formula-is-already-installed/1607

Reply:
Oh that's cool, I didn't know about brew bundle. But seems a little heavy-handed for this. Let's stick with your original for the moment.